webdriver/tests/bidi/__init__.py

```diff
@@ -79,3 +79,25 @@ async def create_console_api_message(bidi_session, context, text):
         target=ContextTarget(context["context"]),
     )
     return text
+
+def remote_mapping_to_dict(js_object):
+    obj = {}
+    for key, value in js_object:
+        obj[key] = value["value"]
+
+    return obj
+
+async def get_viewport_dimensions(bidi_session, context):
```

Comment:
Always use two empty lines as separator for top-level functions. This also applies to other files in this PR.

```suggestion
    return obj


async def get_viewport_dimensions(bidi_session, context):
```